backend/spacewalk-backend.changes

```diff
@@ -1,3 +1,5 @@
+- Fix binary blob corruptions in tradidional config file deployment(bsc#1183864)
```

Comment:
Minor nitpick: leave an space before `(bsc#1183864)` (you need to fix conflicts anyway :-))

Reply:
fixed it